src/nupic/engine/Input.hpp

```diff
@@ -240,7 +240,7 @@ class Input {
   void getInputForNode(size_t nodeIndex, std::vector<T> &input) const;
 
   bool hasIncomingLinks() { return !links_.empty(); }
-
+  bool hasSplitterMap() { return !splitterMap_.empty(); }
```

Comment:
is this part of a (public) API now?

Reply:
Don't worry, this gets removed in the next PR.  I added this so that if someone was using a splitterMap the Link class would make a copy of the SDR rather than share it when propagating buffers..